### ISSUE TITLE
GCpedia header and menu not displaying properly when zooming-in

### ISSUE THREAD

**Issue by LemieuxGen** [gctools-outilsgc/gcpedia#49]
The GCpedia header and menu does not displaying properly when zooming-in. 
Users with visual impairments need to be able to zoom in up to 200% when looking at web pages, and need to be able to use the header and menu options.  

![gcpedia zoom in view](https://cloud.githubusercontent.com/assets/15378977/25851506/0aa3d4bc-3494-11e7-80b8-60dcfe02887e.JPG)

When zooming past 200%, the site should transfer to mobile view.

**Comment by Phanoix** [comment]
I tried something a bit different on production which seems to look better or at least shouldn't break at 200%, can you take a look?

**Comment by LemieuxGen** [comment]
Thanks @Phanoix it's a lot better. The only thing is that there's a bracket <0> showing up that shouldn't be there.
![gcpedia header code](https://cloud.githubusercontent.com/assets/15378977/25856273/89503d3c-34a3-11e7-8898-2b9b67914e73.JPG)

**Comment by LemieuxGen** [comment]
@Phanoix The GCintrante and GCdirectory team are asking about the header issues we're experiencing on GCpedia. Zooming in is still a bit of an issue, where the text, images and menu options overlaps. Is there anything else we can go to improve it? I checked Wikipedia, and they don't seem to be having the same issue when zooming in... (not sure if that helps at all!). 
They are also asking about the site switching over to mobile view with the hamburger menu, hamburger” which never appears when the screen is minimized. Can we fix this?

**Comment by Phanoix** [comment]
The gcpedia once shouldn't be switching over to a mobile hamburger menu since it doesn't have that gcconnex style menu to switch over to that... it should be using the wikipedia mobile view for mobile devices anyways. As for the layout issues and the <0> I'll see what can be done to fix that up.

**Comment by Phanoix** [comment]
is the <0> still there? it's not showing up for me now.

**Comment by LemieuxGen** [comment]
It's not there for me anymore, but it there for some other users. Not sure why. Any thing you want me to ask them that would help to determine what's the issue?

**Comment by Phanoix** [comment]
This will all change with new common topbar and if it gcpedia gets merged into the common account.
